app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -42,22 +42,24 @@ class TSnePlotViewRoute extends React.Component {
         }
      ]
 
+    let cellType = _first(_intersection(_map(this.props.metadata,`label`), this.props.initialCellTypeValues))
```

Comment:
Why `let` and not `const`? Also, is the value of this variable a cell type? I think this name is misleading.

Reply:
Makes sense we can use `const` as we do not want to change the variable value later so a better safer option.

Regarding, name I assumed that it's returning different types of cell type values so that's why. Let me think of another word.